### PR TITLE
fix: remove default supportFile path from scaffold

### PR DESCRIPTION
**Pull request by estrada9166**
- Closes 

### PR Tasks

- [ ] Have tests been added/updated?
- [ ] Has the original issue (or this PR, if no issue exists) been tagged with a release in ZenHub? (user-facing changes only)
- [ ] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [ ] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
- [ ] Have new configuration options been added to the [`cypress.schema.json`](https://github.com/cypress-io/cypress/blob/develop/cli/schema/cypress.schema.json)?

### CODE REVIEW

File: packages/data-context/src/actions/WizardActions.ts [hunk 1]
```diff
@@ -321,17 +321,23 @@ interface E2eScaffoldOpts {
 }
 
 const E2E_SCAFFOLD_BODY = (opts: E2eScaffoldOpts) => {
+  let e2eFields = `specPattern: 'cypress/e2e/**/*.cy.{js,ts}',`
+
+  if (opts.lang === 'ts') {
+    e2eFields = `supportFile: 'cypress/support/e2e.ts',
```
Comment (emilyrohrbough) [inline packages/data-context/src/actions/WizardActions.ts:327]: Why is this default support file value added if `ts` is used? Would it be reasonable to assume the default when TS is not provided to be `cypress/support/e2e.js`?

Comment (lmiller1990) [reply packages/data-context/src/actions/WizardActions.ts:327]: I am a little confused too. What does this PR actually do? It says "remove default supportFile" it's more like "only have a default `supportFile` is the lang is `ts` isn't it?

@estrada9166 can you clarify what this PR is doing?

Comment (lmiller1990) [reply packages/data-context/src/actions/WizardActions.ts:327]: Looks like we can close in favor of https://github.com/cypress-io/cypress/pull/19776/files maybe?

**Comment by estrada9166** [comment]
closing in favor of #19776

**Pull request closed without merging**